src/Renderer/PhpRenderer.php

```diff
@@ -358,20 +337,20 @@ public function render($nameOrModel, $values = null)
             $helper = $this->plugin(ViewModel::class);
             $helper->setCurrent($model);
 
-            $values = $model->getVariables();
+            $variables = $model->getVariables();
             unset($model);
         }
 
         // find the script file name using the parent private method
-        $this->addTemplate($nameOrModel);
-        unset($nameOrModel); // remove $name from local scope
+        $this->addTemplate($templateNameOrModel);
+        unset($templateNameOrModel); // remove $name from local scope
 
         $this->__varsCache[] = $this->vars();
 
-        if (null !== $values) {
-            $this->setVars($values);
+        if (null !== $variables) {
```

Comment:
Are the Yoda conditions still used?! I think not.

Reply:
Yeah - I hate yoda conditions too!